models/organization-extension/src/oidc-sso-config/generator.ts

```diff
@@ -0,0 +1,22 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import type { TOidcSsoConfig } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TOidcSsoConfig>({
+  fields: {
+    id: fake((f) => f.string.uuid()),
+    createdAt: fake(getOlderDate),
+    updatedAt: fake(getNewerDate),
+    authorityUrl: fake((f) => f.internet.url()),
+    clientId: fake((f) => f.string.uuid()),
+    clientSecret: fake((f) => f.internet.password()),
+    isActive: fake((f) => f.datatype.boolean()),
+    logoutUrl: fake((f) => f.internet.url()),
+    sessionTokenExpTimeSeconds: fake((f) => f.number.int()),
```

Comment:
This is currently a range value between 1 hour and 29 days so perhaps we can use that to return a valid number.

The value is in milliseconds so it would be a number between `3600` and `2505600`.

Reply:
updated [7a20bb5](https://github.com/commercetools/test-data/pull/563/commits/7a20bb5f2cc638b3bb35f39890ce7fe42dbba977)